poetry.lock

```diff
@@ -23,9 +23,9 @@ optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
 
 [package.extras]
+tests = ["coverage[toml] (>=5.0.2)", "hypothesis", "pympler", "pytest (>=4.3.0)", "six", "zope.interface"]
```

Comment:
Not sure why the lock file has been updated in this changeset. These changes should be dropped.

Reply:
Will do.